packages/@expo/cli/src/export/resolveOptions.ts

```diff
@@ -39,12 +39,17 @@ export function resolvePlatformOption(
         // Pass through so the more robust error message is shown.
         return platform;
       }
-      throw new CommandError(
-        'BAD_ARGS',
-        `Platform "${platform}" is not configured to use the Metro bundler in the project Expo config, or is missing from the supported platforms in the platforms array: [${exp.platforms?.join(
-          ', '
-        )}].`
-      );
+
+      let error = `Platform "${platform}" is not configured to use the Metro bundler in the project Expo config, or is missing from the supported platforms in the platforms array: [${exp.platforms?.join(
+        ', '
+      )}].`;
+
+      if (platform === 'web') {
+        error +=
+          ' More information: https://docs.expo.dev/guides/customizing-metro/#adding-web-support-to-metro.';
```

Comment:
- use `learnMore()` function for this
- link to https://docs.expo.dev/guides/customizing-metro/#web-support - this anchor feels a bit more stable and brings you to the same place more or less